staking-pool/src/lib.rs

```diff
@@ -127,8 +154,73 @@ impl StakingContract {
         let contract: StakingContract = env::state_read().expect("ERR_READ_CONTRACT_STATE");
         contract
     }
+
+    /// create or update tier min point config
+    pub fn set_tier_point(&mut self, tier: Tier, min_point: U64) {
+        self.config.set_tier_min_point_cfg(tier, min_point as u64);
+    }
+
+    /// Get user point(xKula) amount by account id
+    pub fn get_user_point(&self, account_id: AccountId) -> U64 {
+        let account: Option<UpgradableAccount> = self.accounts.get(&account_id);
+        if account.is_some() {
+            let acc: Account = Account::from(account.unwrap());
+            U128(acc.point) as U64
+        } else {
+            U64(0)
+        }
+    }
+
+    /// Get user staking tier and point by account id
+    pub fn get_user_staking_tier(&self, account_id: AccountId) -> (Tier, U64) {
```

Comment:
Should it be _user_tier_ instead of _user_staking_tier_? The same for all remaining places in this PR.